src/oauth2-mock-server.ts

```diff
@@ -99,6 +105,10 @@ Options:
                     If omitted, 8080 will be used.
                     If 0 is provided, the operating system will assign
                     an arbitrary unused port.
+  -c <cert>         Optional file path to an SSL cert, both cert and key need
```

Comment:
Just a minor typo: Please use a period instead of a comma. Same thing with the `-k <key>` entry.

Reply:
Done!